tools/oesign/main.c

```diff
@@ -468,11 +480,21 @@ static const char _usage_gen[] =
 static const char _usage_sign[] =
     "Usage: %s sign {--enclave-image | -e} ENCLAVE_IMAGE "
     "{--config-file | -c} CONFIG_FILE {--key-file | -k} KEY_FILE\n"
+#if HAS_ENGINE_SUPPORT
+    "{{--engine| -n} ENGINE-NAME {--load-path | -p } ENGINE-LOAD-PATH "
```

Comment:
`ENGINE-NAME`, `ENGINE-LOAD-PATH` and `KEY-ID` still not using snake case as with `CONFIG_FILE` etc.

Also lines 493-496, 534 & 535-536.